package.json

```diff
@@ -26,7 +26,7 @@
     "applicationinsights": "^2.9.6",
     "fast-xml-parser": "^4.1.3",
     "ffc-ahwr-event-publisher": "^1.0.3",
-    "ffc-messaging": "^2.3.4",
+    "ffc-messaging": "^2.10.2",
```

Comment:
An aside, but I wonder if it's worth us moving to fixed versions of dependencies, as discussed here: https://github.com/DEFRA/cdp-node-backend-template/pull/83#discussion_r1930250927